docs/manual/manual_angle_control_mechanisms.rst

```diff
@@ -0,0 +1,648 @@
+
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _manual_angle_control_mechanisms:
+
+********************************
+Angle Based Control Mechanisms
+********************************
+
+================================
+About Position Control
+================================
+Vertiq's position control uses a closed loop PID controller in order to achieve a target position. There are two unique methods that we recommend for setting target displacements, Control Angle and Control Trajectory.
+Also available is a Control Velocity which is driven by our angle controller. More information about angle based velocity control can 
+be found <link to the velocity control page>. TODO: ADD ACTUAL LINK TO VELOCITY CONTROL WHEN IT EXISTS
```

Comment:
This is the only link that does not exist yet. Moving forward with the PR to get the information at least onto a findable branch for people who need it.